cmake/mcuboot.cmake

```diff
@@ -130,7 +130,8 @@ function(zephyr_mcuboot_tasks)
     dt_nodelabel(slot1_partition NODELABEL "slot1_partition" REQUIRED)
     dt_reg_addr(slot1_partition_address PATH ${slot1_partition})
 
-    set(imgtool_args --align 1 --load-addr ${chosen_ram_address} ${imgtool_args})
+    dt_prop(write_block_size PATH "${flash_node}" PROPERTY "write-block-size")
+    set(imgtool_args --align ${write_block_size} --load-addr ${chosen_ram_address} ${imgtool_args})
```

Comment:
fallback needed if write-block-size is not set. the generic spi-nor flash driver does not have this prop and also not allow this prop to be added. If it is not set imgtool will complain, that align is without argument

Reply:
I'd argue we need to allow the generic spi-nor-flash driver to add that property in this case- typically this is done by including the `soc-nv-flash` compatible, but you can also define the property directly

Reply:
added it here: https://github.com/zephyrproject-rtos/zephyr/pull/95152

Reply:
@danieldegrasse This change indeed broke the RAMLOAD build on our side.
I see #95152 is supposed to fix it, but it seems blocked for the time being. Would it be possible to revert this specific change waiting to have a consensus on a way to avoid breaking existing users ?